src/util/BaseLoginRouter.js

```diff
@@ -109,13 +110,15 @@ function (Okta, Backbone, BrowserFeatures, xdomain, RefreshAuthStateController,
         // and then the open tooltip will lose focus and close.
       });
 
-      Okta.$(options.el).append('<div id="okta-sign-in" class="auth-container main-container"></div>');
-      this.el = '#okta-sign-in';
-
       this.appState = new AppState({
         baseUrl: this.settings.get('baseUrl'),
         settings: this.settings
       }, { parse: true });
+
+      this.wrapper = new AuthContainer({appState: this.appState});
```

Comment:
Do we need to save a reference to this? For now, let's just use "var wrapper = ..."